joomla/joomla-cms
# [3.10] Add a 6s timeout before running the finalisation script

Pull Request for Issue #36833 .

### Summary of Changes

Adds a timeout so OPCache could have enough time to do the I/O operations needed.

See https://github.com/joomla/joomla-cms/issues/36833#issuecomment-1020896992 for details.

### Testing Instructions

Update a current 3.10-dev installation which has Zend Opcache enabled and working to the latest 4.1 nightly by using the custom update URL https://update.joomla.org/core/nightlies/next_minor_list.xml or the previously downloaded update package https://developer.joomla.org/nightlies/Joomla_4.1.0-dev-Development-Update_Package.zip .

See https://github.com/joomla/joomla-cms/issues/36833#issuecomment-1020934082 for opcache settings which could be used to reproduce the issue.

### Actual result BEFORE applying this Pull Request

When the files have been updated but (I think) before the database updates, a blank page is shown with an error message in the title (full error message see tool tip):

![j4 1-test-pr-36585_after-update-from-3 10-without-patch_1](https://user-images.githubusercontent.com/7413183/150877284-315e4116-c27a-42e6-a5f7-ba45e0046e05.png)

Tool tip a bit bigger:

![j4 1-test-pr-36585_after-update-from-3 10-without-patch_2](https://user-images.githubusercontent.com/7413183/150877867-96bafa80-007d-4baa-8964-018566af4913.png)

When reloading the page, the update resumes and finishes as usual:

![j4 1-test-pr-36585_after-update-from-3 10-ok](https://user-images.githubusercontent.com/7413183/150877751-167df5df-ff7c-4375-928d-881e2b4c8fb9.png)

### Expected result AFTER applying this Pull Request

No blank page to be reloaded in the middle, the update works as expected.

### Documentation Changes Required

None.
